indico/web/templates/error.html

```diff
@@ -0,0 +1,9 @@
+<div class="error-box">
+    <h1>{{ error_message }}</h1>
+    <p>
+        {{ error_description }}
+    </p>
```

Comment:
This would look nicer in one line, like the `<h1>` but no need to waste time on it.